### DISCUSSION TITLE
How to mark as read as soon as retrieved ?

### DISCUSSION THREAD

**Discussion by Draky50110** [FreshRSS/FreshRSS#5553] [Q&A] [answered]
Hello.

Didn't find the good way : in a feed config, I can set "read" with conditions but I didn't find or understand how to retrieve feed (after a normal "refresh" and mark all items inside "read" on retrieve.

Is it possible ?

Thanks.

**Comment by Alkarex** [comment, marked as answer]
See in configuration / reading:
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/ee531635-90f1-4a3c-8fce-ce604a4059c3)

Can also be set on individual feeds.

**Reply by Draky50110** [reply]
Thanks.

You have checkbox, I have drop-down menu o.O
But seems ok.
